sale_stock_partner_warehouse/models/sale_order.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    warehouse_id = fields.Many2one(
+        compute="_compute_warehouse_id", store=True, precompute=True
```

Comment:
why `precompute`?

Reply:
TO compute the value before the record creation

Reply:
@santostelmo As warehouse_id is defined in sale_stock, you don't need to repeat the properties. Just override _compute_warehouse_id()

Reply:
@santostelmo I know, I mean... why do you need that? I guess it's because the wh is supposed to be already there on create.
If there's a motivation, it's worth to add a comment there.